### ISSUE TITLE
JSON-LD / schema.org for the openEO Hub

### ISSUE THREAD

**Issue by m-mohr** [Open-EO/openeo-hub#10]
We should introduce JSON-LD / schema.org support for the openEO Hub, so that alle data sets that are indexed there can be crawled by Google and others. This will bring people to find our data and potentially use our service.

As we are using STAC for the data discovery, we can benefit from the work going on their. STAC Browser is implementing it already and the actual mapping of data is described here: https://github.com/radiantearth/stac-spec/issues/378

**Comment by christophfriedrich** [comment]
You moved this issue around milestones a lot -- are you now sure it should be "future"? I was almost about to label it "future-proofness" (i.e. as one to tackle soon) because it would be one way of making sure the Hub won't be forgotten entirely.

**Comment by m-mohr** [comment]
Yes, we may inherit some stuff from STAC Browser here at some point though, but that still needs some experiments. Once that works, we may move it on the Agenda again.
